### PR TITLE
W2b cleanup

### PR DESCRIPTION
**Pull request by jonlouie**
## Related issue

Closes: #462

## Description
* Remove .DS_Store files
* Verify exclusion of .DS_store files .gitignore
* Replace INamedSymbol with namespace strings to ensure proper comparison
* Move SolutionHash and ProjectGUID properties to a base class in Web Forms metrics
* Move reference to w2b in CTA.Rules.Update.csproj to CTA.Rules.PortCore.csproj

---
*By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.*

### CODE REVIEW

**Review by saikiranakula-amzn — DISMISSED** [review]

**Review by kanikaul-amazon — APPROVED** [review]

File: tst/CTA.WebForms2Blazor.Tests/Extensions/SemanticModelExtensionTests.cs [hunk 1]
```diff
@@ -117,7 +117,7 @@ public async Task GetNamespacesReferencedByType_Does_Not_Contain_Own_Namespace()
             var classSymbol = model.GetDeclaredSymbol(testClass1Declaration);
             var requiredNamespaces = model.GetNamespacesReferencedByType(testClass1Declaration);
 
-            Assert.False(requiredNamespaces.Contains(classSymbol?.ContainingNamespace));
+            Assert.False(requiredNamespaces.Contains(classSymbol?.ContainingNamespace.ToDisplayString()));
```
Comment (marknfawaz) [inline tst/CTA.WebForms2Blazor.Tests/Extensions/SemanticModelExtensionTests.cs:120]: Can this be null?

**Review by marknfawaz — CHANGES_REQUESTED** [review]
Please make sure DS_Store files are removed

File: tst/CTA.WebForms2Blazor.Tests/Extensions/SemanticModelExtensionTests.cs (hunk 1 above)
Comment (comsemer) [reply tst/CTA.WebForms2Blazor.Tests/Extensions/SemanticModelExtensionTests.cs:120]: I don't think we want to allow `classSymbol.ContainingNamespace` to be null here, as we want to assume that any class symbol we are processing belongs to a namespace, so we can probably get rid of the null conditional operator here, though I don't think `classSymbol.ContainingNamespace` should evaluate to null in the first place

**Review by comsemer — COMMENTED** [review]
Looks good to me, may just want to remove the null conditional operator near Mark's comment

File: tst/CTA.WebForms2Blazor.Tests/Extensions/SemanticModelExtensionTests.cs (hunk 1 above)
Comment (jonlouie) [reply tst/CTA.WebForms2Blazor.Tests/Extensions/SemanticModelExtensionTests.cs:120]: According to docs, ContainingNamespace can be null. It's possible to define classes without a namespace block. I believe in those cases, ContainingNamespace would be null.

Comment (comsemer) [reply tst/CTA.WebForms2Blazor.Tests/Extensions/SemanticModelExtensionTests.cs]: Ah ok, makes sense, thanks for clearing that up

**Review by comsemer — APPROVED** [review]
Looks good to me

**Review by marknfawaz — APPROVED** [review]

**Pull request merged**